core/src/test/scala/unit/kafka/integration/MetricsDuringTopicCreationDeletionTest.scala

```diff
@@ -70,12 +70,12 @@ class MetricsDuringTopicCreationDeletionTest extends KafkaServerTestHarness with
   @Test
   def testMetricsDuringTopicCreateDelete(): Unit = {
 
-    // For UnderReplicatedPartitions, because of https://issues.apache.org/jira/browse/KAFKA-4605
+    // For UnderReplicatedPartitions, because of KAFKA-4605
```

Comment:
Why are we removing the links?

Reply:
I just thought the ticket Id: KAFKA-4605 is enough, no need to put  the whole URL. What do you think?

Reply:
I just reverted my change to put whole URL here. I think either way is OK. Thanks.